src/dashmap_impl.rs

```diff
@@ -44,15 +44,15 @@ impl<'a, T: ?Sized> ArcRef<'a, T> {
     }
 }
 
-impl<K: Hash + Eq, V: ?Sized> Default for CacheMap<K, V> {
+impl<K: Hash + Eq, V: ?Sized, S: BuildHasher + Default + Clone> Default for CacheMap<K, V, S> {
     fn default() -> Self {
         CacheMap {
             inner: Default::default(),
         }
     }
 }
 
-impl<K: Hash + Eq, V> std::iter::FromIterator<(K, V)> for CacheMap<K, V> {
+impl<K: Hash + Eq, V> std::iter::FromIterator<(K, V)> for CacheMap<K, V, RandomState> {
```

Comment:
```suggestion
impl<K: Hash + Eq, V, S: BuildHasher + Default + Clone> std::iter::FromIterator<(K, V)> for CacheMap<K, V, S> {
```

Reply:
@afranchuk `DashMap` of version `4.0.2` (used by this crate) can only be built from iterator with `RandomState` hasher: https://docs.rs/dashmap/4.0.2/dashmap/struct.DashMap.html#impl-FromIterator%3C(K%2C%20V)%3E

I shall update the version to the latest (https://docs.rs/dashmap/5.5.3/dashmap/struct.DashMap.html#impl-FromIterator%3C(K,+V)%3E-for-DashMap%3CK,+V,+S%3E) where it is supported